documentation/sphinx/source/release-notes/release-notes-720.rst

```diff
@@ -3,12 +3,20 @@
 #############
 Release Notes
 #############
+71.2.6
+======
+
+Features
+--------
+- Read-aware Data Distribution is enabled by default. ``knob_READ_SAMPLING_ENABLED=true`` by default.
```

Comment:
Nit: Should knob_ be capitalized? (Also in the command-line-interface.rst file)

Reply:
I think in `foundationdb.conf` we write `knob_` prefix in lower-case